workspaces/adventure-pack/goodies/typescript/Iterator.prototype.take/index.ts

```diff
@@ -0,0 +1,32 @@
+import "../Iterator.prototype.toIterable";
+import { iteratorPrototype } from "../Iterator.prototype";
+
+declare global {
+  interface Iterator<T> {
+    take(this: Iterator<T>, count: number): IterableIterator<T>;
```

Comment:
Let's call it limit, like MDN does: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/take#limit

(here and below)